ouroboros-consensus-cardano/test/cardano-test/Test/Consensus/Cardano/Show.hs

```diff
@@ -0,0 +1,9 @@
+module Test.Consensus.Cardano.Show () where
+
+import           Ouroboros.Consensus.Cardano ()
+import           Ouroboros.Consensus.Cardano.Block
+import           Ouroboros.Consensus.Shelley.Ledger.SupportsProtocol ()
```

Comment:
Would be really nice to get rid of the orphan instances (#588), but this is out of scope for this PR